css/store.css

```diff
@@ -0,0 +1,144 @@
+
+ /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+
+ body {
+    font-family: 'Roboto', sans-serif;
+    -webkit-font-smoothing: antialiased;
+}
+
+/**
+ * Add your custom styles below
+ * 
+ * Remember: 
+ * - Be organised, use comments and separate your styles into meaningful chunks
+ *    for example: General styles, Navigation styles, Hero styles, Footer etc.
+ * 
+ * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
+ */
+
+ body {
+    margin: 0;
+  }
+  
+  body,
+  button {
+    color: var(--grey-dark);
+    font-size: 1rem;
+  }
+  
+  h1,h2,h3,h4,h5,h6 {
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+  
+  p {
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+  
+  .header,
+  .content,
+  .footer {
+    padding: 0 1.5rem;
+    margin: 0 auto;
+  }
+/* Header */
+ .header {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    height: 5rem;
+    margin-bottom: 1rem;
+    padding-top: 1rem;
+  }
+  
+  .header__logo {
+    max-height: 40px;
+    width: 40px;
+  }
+  
+/* Navigation */
+
+.navigation__list {
+    display: flex;
+    list-style: none;
+    gap: 20px;
+  }
+  
+  .navigation__item {
+    padding: 0.5rem 0;
+  }
+  
+  .navigation__link {
+    color: var(--grey-dark);
+    font-weight: 300;
+    text-decoration: none;
+  }
+  
+  .navigation__link:hover {
+    color: #c05326;
+  }
+
+/* karma-store */
+.content-store {
+    display: flex;
+    justify-content: space-evenly;
+}
+
+.article__summary {
+    font-size: 50px;
+    font-weight: 500;
+    color: #c05326;
+}
+/* contact-form */
+.orderForm{
+    display : flex;
+    flex-direction : column;
+    width: 501px;
+    height: 1200px;
+    left: 122px;
+    background-color: aquamarine;
+  }
+
+.orderForm h2 {
```

Comment:
https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/

this is a good reading about class naming convention. I was discovering myself as well while sending it 😄